### PR TITLE
Assist on Android: how to make it the default assistant

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Assist on Android: 
- Add procedure how to make Assist the default assistant on an Android phone and on Wear OS
- Update procedures on setting up and using Wear OS to reflect current software
- Add procedure on adding Assist as a complication to the Watch face.

## Type of change

- [] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/voice_control/android.markdown [hunk 1]
```diff
@@ -2,18 +2,45 @@
...
+1. On Android, go to **Settings** > **Apps** > **Default apps**.
+2. Under **Digital assistant app**, select **Default digital assistant app**.
+   - On some Android versions, this might be under **Digital & voice input** > **Default digital assistant app** > **Voice Assistant**.
+3. Select **Home Assistant**.
+   - On some Android versions, you might be able to change the assistants language settings here by selecting the cogwheel.
+4. Go back one step. The **Default digital assistant app** should now show *Home Assistant* as the default.
+5. Leave the **Settings**.
+6. Start Assist using the gesture to start an assistant. The gesture may differ depending on the version.
+   - Swipe from the bottom left corner.
+   - Long press the power button.
```
Comment (synesthesiam) [inline source/voice_control/android.markdown:28]: For my phone, it was holding the home button (square button at the bottom)

File: source/voice_control/android.markdown [hunk 2]
```diff
@@ -2,18 +2,45 @@
...
+## Assist on Wear OS
+
 Assist is available on Wear OS using the [Home Assistant Companion App](https://companion.home-assistant.io/) for Android and "Assist" tile.
 
 <lite-youtube videoid="Dr_ZCbt8w5k" videotitle="Assist on Wear OS"></lite-youtube>
 
-## Installation
+### Installation
 
 After [installing the companion app](https://companion.home-assistant.io/docs/getting_started/) and connecting it to your Home Assistant, visit your watch companion app or the Wear OS app and click "Add tile" inside the Tiles area. Select the "Assist" tile to add it to your watch:
```
Comment (synesthesiam) [inline source/voice_control/android.markdown]: It's also possible to add a "complication" within the watch face editing screen.

Plus, you can set Assist as the default digital assistant and hold the secondary button to bring it up (even if the watch is asleep).

File: source/voice_control/android.markdown (hunk 1 above)
Comment (c0ffeeca7) [reply source/voice_control/android.markdown:28]: Thank you for the hint! Added information.

File: source/voice_control/android.markdown (hunk 2 above)
Comment (c0ffeeca7) [reply source/voice_control/android.markdown:39]: Could we do the Wear OS procedure in a separate PR? I haven't started on that yet.

**Review by synesthesiam — DISMISSED** [review]

File: source/voice_control/android.markdown (hunk 2 above)
Comment (c0ffeeca7) [reply source/voice_control/android.markdown]: Thank you for the input! Added separate procedures on this.

**Review by synesthesiam — APPROVED** [review]

**Pull request merged**